Doc/reference/compound_stmts.rst

```diff
@@ -243,9 +243,8 @@ is found that matches the exception.  An expression-less except clause, if
 present, must be last; it matches any exception.  For an except clause with an
 expression, that expression is evaluated, and the clause matches the exception
 if the resulting object is "compatible" with the exception.  An object is
-compatible with an exception if the object is the class or a base class of the exception
-object, or a tuple containing an item that is the class or a base class of
-the exception object.
+compatible with an exception if the object is the class or a non-virtual base
+class of the exception object, or a tuple containing one or more such items.
```

Comment:
I deleted my comment about this part of the wording, the original text with the singular was not incorrect.
(I understand it to mean: the clause can match **one** of the exception classes in the tuple)

Reply:
Do you think I should revert my last change here? I was on the fence on it, both old and new seem correct but I'm not sure which is easier to read.

Reply:
I agree with reverting.  I also think it wouldn’t be that bad to repeat «a tuple containing an item that is the class or a non-virtual base class of the exception object»; this is the Language Reference, and we don’t want people to be puzzled over «item».